api/api.proto

```diff
@@ -3447,6 +3446,15 @@ message NodeDrainAttachmentsJob {
     google.protobuf.Timestamp last_update_time = 6;
 }
 
+message CloudDriveTransferJob {
+  // SourceNodeID is the ID of the node whose cloud driveset needs to be transferred
+  string source_node_id = 1;
```

Comment:
Should this be instance id or scheduler node name? 
For a user this will be a bit convoluted since the source node id is the ID that the destination instance will latch onto.

Reply:
renamed to source_drive_set and destination_drive_set and updated the variable doc.